glitter/reminders/management/commands/send_reminders.py

```diff
@@ -0,0 +1,55 @@
+from django.conf import settings
+from django.contrib.sites.models import Site
+from django.core.mail import send_mail
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from glitter.reminders.models import Reminder
+
+
+class Command(BaseCommand):
+    help = 'Management command to send reminder about out dated content.'
+
+    def handle(self, *args, **options):
+        for reminder in Reminder.objects.select_related('content_type').iterator():
+            content_obj = reminder.content_object
+
+            if content_obj.current_version and content_obj.published:
+
+                date_difference = timezone.now() - content_obj.current_version.modified
+
+                # Check when the last time this reminder was sent to make sure it's not sending
+                # every day to remind it.
+                if reminder.sent_at:
+                    date_difference = timezone.now() - reminder.sent_at
```

Comment:
I don't think the logic for this one is right, as once this is set it'll just spam users every interval, regardless of their published date.

Might be easier to talk through this one.